openedx/features/job_board/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+
+from django_countries.fields import CountryField
+from model_utils.models import TimeStampedModel
+
+from .constants import JOB_COMPENSATION_CHOICES, JOB_HOURS_CHOICES, JOB_TYPE_CHOICES
+
+
+class Job(TimeStampedModel):
+    """
+    This model contains all the fields related to a job being
+    posted on the job board.
+    """
+    title = models.CharField(max_length=255)
+    company = models.CharField(max_length=255)
+    type = models.CharField(max_length=255, choices=JOB_TYPE_CHOICES)
```

Comment:
As we are clear with max possible length from choices, do you think we should still set it to `255`.

Reply:
Please refer to this: https://stackoverflow.com/questions/50530794/why-should-i-set-max-length-when-using-choices-in-a-django-model